game_eggs/steamcmd_servers/life_is_feudal_your_own/egg-life-is-feudal--your-own--expect.json

```diff
@@ -0,0 +1,131 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-08-23T15:02:06+02:00",
+    "name": "Life is Feudal: Your Own - Expect",
+    "author": "tueye@tuworld.de",
+    "description": "Life is Feudal is an open persistent virtual world, allowing players can roam around freely without restrictions. Its terraforming abilities allow you to shape the terrain organized in a unique grid-pattern. This means that players can not only raise, lower and flatten terrain, but also dig tunnels and underground caverns. The game also includes a skill system, mini-games, loot, PvP, and crafting.",
+    "features": null,
+    "images": [
+        "ghcr.io\/tueye\/pt-images_wine:wine-expect"
```

Comment:
why not use an image from my repo?

Reply:
Or maybe PR an updated image to the repo.

Reply:
> 
> 
> Or maybe PR an updated image to the repo.

Do you mean a PR for the parkervcp/yolks repo with a new image or a PR for parkervcp/images with a change for the existing debian/wine image?